test/pangolin/pangolin.test.js

```diff
@@ -0,0 +1,204 @@
+const { expect } = require("chai");
+const hre = require("hardhat");
+
+const { web3, deployments, waffle, ethers } = hre;
+const { provider, deployContract } = waffle
+
+const abis = require("../../scripts/constant/abis");
+const buildDSAv2 = require("../../scripts/buildDSAv2_avalanche")
+const deployAndEnableConnector = require("../../scripts/deployAndEnableConnector.js")
+const encodeSpells = require("../../scripts/encodeSpells.js")
+const getMasterSigner = require("../../scripts/getMasterSigner_avalanche")
```

Comment:
update them as well.
Some things here are not needed as the repo has been updated to support multichain support !